style.css

```diff
@@ -447,11 +480,20 @@ div.data-dashbord div.button button.html {
 }
 
 .container-projects-grid {
-  margin-top: 2.4rem;
+  width: 90%;
+  margin: auto;
   display: grid;
-  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
-  grid-template-rows: auto;
-  gap: 2.4rem;
+  margin-top: 2.4rem;
+  grid-template-columns: repeat(auto-fit, minmax(340px, 369px));
+
+  /* grid-template-rows: auto; */
+  row-gap: 1rem;
+
+  /* justify-content: center; */
+
+  /* align-content: center; */
```

Comment:
- [ ] I kindly suggest to avoid leaving commented code in your code base. This is because Leaving commented-out code in source code is bad practice, as it takes up space, causes confusion, and leads to maintenance issues if it's not removed 😞.